MovieFestival/Movie.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MovieFestival
+{
+	internal class Movie
+	{
+		public int Length { get; set; }
+		public string Genre { get; set; }
+		public string Title { get; set; }
+
+		public Movie(int length, string genre, string title)
+		{
+			Length = length;
+			Genre = genre;
+			Title = title;
+		}
+
+		public string GetData()
+		{
+			string movie = $"{Title} - {Length} minutes - {Genre}";
```

Comment:
Make sure that output is in format ex `Batman, 123min, DA` `Title, Length, Genre`

Reply:
Output format is changed to Title, Length, Genre